cf-testd/cf-testd.c

```diff
@@ -509,8 +529,6 @@ int CFTestD_StartServer(CFTestD_Config *config)
 
     int sd = InitServer(CFTESTD_QUEUE_SIZE, config->address);
 
-    MakeSignalPipe();
-
     int selected = 0;
     while (selected != -1)
```

Comment:
`WaitForIncoming()` returns `-2` for interrupt, so maybe check for this also?

Reply:
I think we just want to retry in such cases.